lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -78,22 +86,23 @@ def post_notes(app_id, release_id, release_notes)
         UI.success("✅ Posted release notes.")
       end
 
-      # Get app
+      # Get AAB info (Android apps only)
       #
       # args
       #   app_id - Firebase App ID
       #
       # Throws a user_error if the app hasn't been onboarded to App Distribution
-      def get_app(app_id, app_view = 'BASIC')
+      def get_aab_info(app_id)
         begin
-          response = connection.get("#{v1_apps_url(app_id)}?appView=#{app_view}") do |request|
+          aab_name = aab_info_name(app_id)
+          response = connection.get("#{aab_info_url(app_id)}?name=#{aab_name}") do |request|
```

Comment:
I don't _think_ you need the `name` query parameter. The backend should be able to extract it from the URL.

Reply:
Good call, removed.